src/hooks/use-keyboard-controls.js

```diff
@@ -33,14 +36,35 @@ const useKeyboardControls = ({
             e.preventDefault();
           }
         }
+        if (!!e.altKey) {
```

Comment:
I guess there may be an argument that this hook is doing too much. 

I like that all the logic is co-located but happy for feedback.